common/src/main/java/tachyon/network/protocol/RPCErrorResponse.java

```diff
@@ -44,17 +40,25 @@ public static RPCErrorResponse decode(ByteBuf in) {
   }
 
   @Override
-  public int getEncodedLength() {
-    // 1 short (mStatus)
-    return Shorts.BYTES;
+  public void encode(ByteBuf out) {
+    out.writeShort(mStatus.getId());
   }
 
   @Override
-  public void encode(ByteBuf out) {
-    out.writeShort(mStatus.getId());
+  public int getEncodedLength() {
+    // 1 short (mStatus)
+    return Shorts.BYTES;
   }
 
   public Status getStatus() {
     return mStatus;
   }
+
+  public Type getType() {
+    return Type.RPC_ERROR_RESPONSE;
+  }
+
+  public String toString() {
+    return mStatus.getMessage();
```

Comment:
@gpang you mean we should return the type of RPCErrorResponse(aka message) instead of the status(aka response)? in this case, shall I also override the toString method of the enum [Type](https://github.com/amplab/tachyon/blob/master/common/src/main/java/tachyon/network/protocol/RPCMessage.java#L33) to print something meaningful? Of course another option is to wrap it like this:
```
RPCErrorResponse.toString() {return "RPCErrorResponse(" + getType().toString() + ")"}
```